gunpowder/jax/generic_jax_model.py

```diff
@@ -0,0 +1,14 @@
+
+class GenericJaxModel():
+
+    def __init__(self):
+        pass
+
+    def initialize(self, rng_key, inputs, is_training):
+        raise RuntimeError("Unimplemented")
+
+    def forward(self, params, inputs):
+        raise RuntimeError("Unimplemented")
+
+    def train_step(self, params, inputs, pmapped=False):
```

Comment:
expected outputs should probably be documented. Implementation makes it look like you must return `self.model_params, outputs, loss`. Where outputs is `Dict[Any, np.ndarray]`. I assume loss is just a float and there is some reasonable default storage for jax model parameters?

Reply:
Yes will add more documentations